ansible/library/minigraph_facts.py

```diff
@@ -353,6 +352,10 @@ def parse_xml(filename, hostname):
     if hwsku == "Force10-S6000":
         for i in range(0, 128, 4):
             port_alias_map["fortyGigE0/%d" % i] = "Ethernet%d" % i
+    elif hwsku == "Force10-S6100":
```

Comment:
The list of SKUs seems to be hard coded.
Wouldn't it be better to have it read from some file which contains 
mapping: <sku_name, sku_handler.py> and then invoke the handler?